tools/building.py

```diff
@@ -1232,6 +1232,27 @@ def run_wasm_opt(infile, outfile=None, args=[], **kwargs):  # noqa
   return run_binaryen_command('wasm-opt', infile, outfile, args=args, **kwargs)
 
 
+def run_wasm_bindgen(infile, outfile=None, args=[], **kwargs):  # noqa
+  if not os.path.exists(infile):
+    exit_with_error('wasm-bindgen: wasm file not found (%s).' % infile)
+
+  cmd = config.WASM_BINDGEN + [
+    infile,
+    '--target',
+    'emscripten',
+    '--keep-lld-exports',
+    '--out-dir',
+    get_emscripten_temp_dir() + '/wbg_out'
+  ]
+  ret = check_call(cmd).stdout
```

Comment:
My default `check_call` won't capture stdout but send it to the terminal (which is probably what you want)?   So I think you can remove `ret =` and `.stdout` here.